tests/Jobs/SendEmptyWikibaseNotificationsJobTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\Jobs\SendEmptyWikibaseNotificationsJob;
+use App\Notifications\EmptyWikibaseNotification;
+use App\User;
+use App\Wiki;
+use App\WikibaseNotificationSentRecord;
+use App\WikiLifecycleEvents;
+use App\WikiManager;
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Notification;
+use Tests\TestCase;
+use function Amp\Promise\first;
+
+class SendEmptyWikibaseNotificationsJobTest extends TestCase
+{
+    use RefreshDatabase;
+
+    // the job does not fail in general
+    public function testEmptyWikibaseNotifications_Success()
+    {
+        $mockJob = $this->createMock(Job::class);
+        $mockJob->expects($this->never())
+                ->method('fail')
+                ->withAnyParameters();
+        $job = new SendEmptyWikibaseNotificationsJob();
+        $job->setJob($mockJob);
+        $job->handle();
+    }
+
+    // empty wikis, that are older than 30 days, trigger a notification
+    public function testEmptyWikibaseNotifications_SendNotification()
+    {
+        Notification::fake();
+        $user = User::factory()->create(['verified' => true]);
+        $wiki = Wiki::factory()->create(['created_at' => '2022-12-31 16:00:00']);
```

Comment:
This is a personal preference, so feel free to disregard: I think it's easier to read if you construct a date like this:
```php
Wiki::factory()->create(['created_at' => Carbon::now()->subDays(100));
```
This way I can instantly understand the wiki is 100 days old when the test runs. Right now, I need to do some mental gymnastics to figure out whether the given date is long ago or not.